DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/README.md

```diff
@@ -0,0 +1,315 @@
+
+![](README.md.assets/csr-output-example-simple.svg)
+
+This example is intended for users interested in creating standalone modules that can be included in Intel® Quartus® Prime projects. It serves as a minimal example, and while it targets a very specific board, a user familiar with the Intel® Quartus® Prime suite should be able to easily port this design to other hardware.
+
+### Board-specific Considerations
+
+This design is intended to work with the Intel® Arria® 10 SX SoC Development Kit. The board specific configurations are:
+1. Choose `10AS066N3F40E2SG` device to match the devkit
+2. Choose pin `PIN_AM10` to drive the `i_clk` signal
+3. Choose pin `PIN_AR23` to drive the `fpga_led` signal
+4. Choose pin `PIN_AV21` to drive the `reset_button_n` signal
+5. Use `jtag.sdc` from the Intel® Arria® 10 SoC Golden Hardware Reference Design (GHRD) [source code](https://github.com/altera-opensource/ghrd-socfpga).
+
+## Building the `platform_designer` Tutorial
+
+> **Note**: When working with the command-line interface (CLI), you should configure the oneAPI toolkits using environment variables. 
+> Set up your CLI environment by sourcing the `setvars` script located in the root of your oneAPI installation every time you open a new terminal window. 
+> This practice ensures that your compiler, libraries, and tools are ready for development.
+>
+> Linux*:
+> - For system wide installations: `. /opt/intel/oneapi/setvars.sh`
+> - For private installations: ` . ~/intel/oneapi/setvars.sh`
+> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
+>
+> Windows*:
+> - `C:\Program Files(x86)\Intel\oneAPI\setvars.bat`
+> - Windows PowerShell*, use the following command: `cmd.exe "/K" '"C:\Program Files (x86)\Intel\oneAPI\setvars.bat" && powershell'`
+>
+> For more information on configuring environment variables, see [Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html) or [Use the setvars Script with Windows*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+Follow these steps to compile and test the design:
+1. Compile the SYCL code to RTL using a Windows or Linux machine. 
+
+   Linux:
+
+   ```bash
+   $> cd add-oneapi
+   $> mkdir build
+   $> cd build
+   $> cmake ..
+   $> make fpga_ip_export
+   ```
+
+   Windows:
+
+   ```bash
+   > cd add-oneapi
+   > mkdir build
+   > cd build
+   > cmake -G "NMake Makefiles" ..
+   > nmake fpga_ip_export
+   ```
+
+   For more details, see the README in the `add-oneapi` directory.
```

Comment:
Is it necessary to have two levels of READMEs?
The purpose of this tutorial is to show the quartus integration, no to run emulation/simulation etc.
Would it make sense to remove the inner README entirely?
These commands you are showing here are enough for generating the IP, which is what we are trying to achieve

Reply:
No, i prefer to leave the inner readme so it's clear that you can emulate/simulate the design as well.

Reply:
Then you can add the instructions in this readme, this should all be minimal anyway

Reply:
ok